### PR TITLE
disk size

### PR DESCRIPTION
**Pull request by pacogomez**
fix disk size param and examples

### CODE REVIEW

File: vcd_cli/disk.py [hunk 1]
```diff
@@ -41,13 +42,16 @@ def disk(ctx):
         vcd disk info disk1 --id 91b3a2e2-fd02-412b-9914-9974d60b2351
             Get details of the disk named 'disk1' that has the supplied id.
 \b
-        vcd disk create disk1 100 --description '100 MB Disk'
-            Create a new 100 MB independent disk named 'disk1' using the default storage profile.
+        vcd disk create disk1 1000 --description '1 KB Disk'
```
Comment (rocknes) [inline vcd_cli/disk.py]: I think 
1 KB = 1024 B.
1kB = 1000 B

File: vcd_cli/vdc.py [hunk 2]
```diff
@@ -124,3 +124,22 @@ def use(ctx, name):
         raise Exception('not found')
     except Exception as e:
         stderr(e, ctx)
+
+
+@vdc.command(short_help='create a virtual datacenter')
```
Comment (rocknes) [inline vcd_cli/vdc.py]: Why are we adding dead code?

File: vcd_cli/disk.py (hunk 1 above)
Comment (pacogomez) [reply vcd_cli/disk.py]: so the notation is KiB for 1024 and KB for 1000

File: vcd_cli/vdc.py (hunk 2 above)
Comment (pacogomez) [reply vcd_cli/vdc.py]: wip, I will remove

**Pull request merged**